test/sequential/test-debugger-debug-brk.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
 const common = require('../common');
 common.skipIfInspectorDisabled();
-const fixtures = require('../common/fixtures');
+
+// This test ensures that the debug-brk flag will spin up a new process and
+// wait, aka not quit.
```

Comment:
Micro-nit that you can ignore if you wish: `aka not quit` may be a bit unclear to people unfamiliar with a.k.a. Might be better as `...and wait, rather than exit.`